app/models/user.rb

```diff
@@ -1,17 +1,17 @@
 class User < ActiveRecord::Base
   has_many :dreams
   has_many :keywords, through: :dreams
+  after_create :first_dream
 
   mount_uploader :avatar, AvatarUploader
-  after_create :first_dream
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
+  private
+
   def first_dream
-    @dream = Dream.create(user: self, date: Date.today, sentiment: "0.8", title: "Welcome to your Dream Journal", text: "This is where your dreams are stored! Scroll through your journal timeline to see the details of each dream.", mixed: false)
-    @dream.save
-    @dream.keyword_analysis
+    Dream.create(user: self, date: Date.today, sentiment: "0.8", title: "Welcome to your Dream Journal", text: "This is where your dreams are stored! Scroll through your journal timeline to see the details of each dream.")
```

Comment:
Line is too long. [222/80]<br>Do not use `Date.today` without zone. Use `Time.zone.today` instead.